src/components/ProposalList/ProposalListItem.tsx

```diff
@@ -1,58 +1,55 @@
-import { Avatar, AvatarImage, AvatarFallback } from '@radix-ui/react-avatar';
 import { Label } from '@radix-ui/react-label';
 import { RadioGroup, RadioGroupItem } from '@radix-ui/react-radio-group';
 import { ThumbsUpIcon, ThumbsDownIcon, MinusIcon } from 'lucide-react';
-import { CardHeader, CardContent, Card } from './ui/card.tsx';
+import {
+  CardHeader,
+  CardContent,
+  Card,
+  CardTitle,
+  CardDescription,
+} from '../ui/card.tsx';
+import type { Proposal } from '../../types/proposal.ts';
 
-export type ProposalVoteCardProps = {
-  author?: string;
-  title: string;
-  description: string;
-  created: string;
+export type Props = {
+  proposal: Proposal;
 };
 
-export function ProposalVoteCard(props: ProposalVoteCardProps) {
+export default function ProposalListItem({ proposal }: Props) {
   return (
     <Card>
       <CardHeader className="pb-0 pt-6">
-        <div className="flex items-center justify-between">
-          <div>
-            <h3 className="text-lg font-semibold dark:text-gray-200">
-              {props.title}
-            </h3>
-            <p className="text-gray-500 dark:text-gray-400">
-              {props.description}
-            </p>
-          </div>
+        <CardTitle className="flex justify-between">
+          {proposal.title}
+
           <div className="flex items-center gap-2">
             <ThumbsUpIcon className="w-5 h-5 text-green-500" />
             <span className="text-green-500 font-medium">72</span>
             <ThumbsDownIcon className="w-5 h-5 text-red-500" />
             <span className="text-red-500 font-medium">18</span>
           </div>
```

Comment:
ISSUE: This probably shouldn't be part of the card title
SUGGESTION: Let's use `CardTitle` only around the title prop